Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/Pages/tagsList.html

```diff
@@ -0,0 +1,9 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<ul class="tagsList">
+    <f:for each="{items}" as="item">
```

Comment:
Why is this template in Pages? Is that a separate page?
I would not use f:for so often and resort more on TS... But do more as you see fit.